wys/api/python/wys_api.py

```diff
@@ -478,9 +467,15 @@ def get_schedules(conn, api_key):
                          headers=headers)
     schedule_list = response.json()
```

Comment:
My comment was a bit slow.

And realizing, given the response you output, that it wouldn't handle this specific error. But I think we should also add the `raise_for_status()` here. https://github.com/CityofToronto/bdit_data-sources/issues/713#issuecomment-1741418875